hwdb.d/60-keyboard.hwdb

```diff
@@ -1282,13 +1282,17 @@ evdev:atkbd:dmi:bvn*:bvr*:bd*:svnMicro-Star*:pn*A10SC*:*
 
 # MSI Modern series
 evdev:atkbd:dmi:bvn*:bvr*:bd*:svnMicro-StarInternational*:pnModern*:*
+ KEYBOARD_KEY_56=backslash                              # secondary backslash key
  KEYBOARD_KEY_f1=f20                                    # Fn+F5 micmute
  KEYBOARD_KEY_76=f21                                    # Fn+F4 touchpad, becomes meta+ctrl+toggle
  KEYBOARD_KEY_91=prog1                                  # Fn+F7 Creation Center, sometime F7
  KEYBOARD_KEY_f2=prog2                                  # Fn+F12 screen rotation
+ KEYBOARD_KEY_8d=prog3                                  # Fn+a
+ KEYBOARD_KEY_8c=prog4                                  # Fn+z
```

Comment:
how are these keys labelled?

Reply:
https://download.msi.com/archive/mnu_exe/nb/MS-14DK_v2.0_English.pdf#page=27

They aren't labelled anything on the keyboard 
Should I add the name of application that it triggers in windows?

Reply:
yeah, a quick comment saying briefly what this does would probably make sense, i.e. "on Windows, this spawns xyz" or o.